crypto/cipher/errors.go

```diff
@@ -0,0 +1,17 @@
+package cipher
+
+import (
+	"github.com/pkg/errors"
+)
+
+// ErrEncrypt returns the error message if encryption failed
+//
+func ErrEncrypt() error {
+	return errors.New("cipher: encryption failed")
+}
```

Comment:
For the error values I think global variables would be best in this case 
```
var (
	ErrEncrypt = errors.New("cipher: encryption failed") //nolint:gochecknoglobals
	ErrDecrypt = errors.New("cipher: decryption failed") //nolint:gochecknoglobals
)
```
would work well

Reply:
Thanks, I change this.